Header header

geometry_msgs/Point position
uint8 type # 0 means base, 1 means battery, 2 means pot